README.md

```diff
@@ -0,0 +1,62 @@
+# background checks
+An example Temporal Application
+
+## What do I need to have installed to run the application?
+
+Make sure you have the following installed:
+
+- [Go](https://go.dev/dl/) (latest stable version is recommended)
```

Comment:
Go isn't necessarily required. If they are always using `./start` to run everything via Docker then all the compilation happens in the containers.